docs/architecture/modernize-desktop/migrate-modern-applications.md

```diff
@@ -50,13 +46,13 @@ The <xref:Microsoft.Extensions.Configuration.ConfigurationBuilder> object lets y
 
 ### Migrating Configuration files
 
-You can continue to use your existing app.config XML file. However, you could take this opportunity to migrate your configuration to benefit from the several enhancements made on .NET Core.
+You can continue to use your existing app.config XML file. However, you could take this opportunity to migrate your configuration to benefit from the several enhancements made on .NET.
 
 To migrate from an old-style *app.config* to a new configuration file, you should choose between an XML format and a JSON format.
 
 If you choose XML, the conversion is straightforward. Since the content is the same, just rename the *app.config* file to a file with XML extension. Then, change the code that references AppSettings to use the `ConfigurationBuilder` class. This change should be easy.
```

Comment:
I'm curious as to why the name change is required at all, technically you could instruct your configuration builder to rely on XML and load a file named as it was.

Reply:
Sorry. I thought it was a generic question, and that's the reason I missed it :blush:
Yes, the `rename` word doesn't sound right in that context. Because we just need to save the file type as `XML` type. We can keep the file name as is like `app.config`. How about we modify the sentence something like `just save the *app.config* file with XML as type` ? Because after this step, it would look like :

![image](https://user-images.githubusercontent.com/13281246/103979774-7edf9380-51a4-11eb-9ed0-37842e689f64.png)

![image](https://user-images.githubusercontent.com/13281246/103979798-8e5edc80-51a4-11eb-8451-c05fe999409b.png)

Reply:
Hi @IEvangelist - Could you please share your thought on this ? Thanks.

Reply:
I'm good with it. Thanks